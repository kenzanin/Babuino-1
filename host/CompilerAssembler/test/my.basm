	.global _start

	
	.data
foo:	.byte		true
	
	.text
_start:
	global		foo
	qget
	block
	beep
	eob
	if
	exit
	
	.end
