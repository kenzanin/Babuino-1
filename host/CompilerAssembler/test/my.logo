make "foo true

if :foo
[
	beep
]
