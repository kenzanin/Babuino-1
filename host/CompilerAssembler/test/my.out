146
20
121
138
0
0
0
0
123
143
0
0
2
0
1
146
4
142
0
28
120
124
143
0
16
152
35
96
0
146
11
121
1
3
2
0
1
144
0
0
148
37
1
5
2
0
2
144
0
0
148
37
1
7
2
0
3
144
0
0
148
37
91
136
3
232
1
4
144
0
4
1
4
144
0
0
145
0
2
149
36
93
95
138
0
0
0
0
152
28
3
0
53
144
0
8
2
0
0
2
0
3
2
0
1
3
0
36
144
0
8
149
36
144
0
4
148
38
144
0
10
148
35
144
0
10
148
36
168
144
0
8
149
36
145
0
4
157
36
153
37
4
126
4
10
95
152
8
7
7
